javascript/codefight/extractEachKth.js

```diff
@@ -1,5 +1,5 @@
 export const extractEachKth = (inputArray, k) => {
-    let outPut = [];
+    const outPut = [];
```

Comment:
This array is initially empty, and then filled. The value of a constant cannot change through re-assignment, and it can't be redeclared.

Reply:
In JS you can declare array with const and push items in it. It will only throw an error if you will try to re-assign variable.